storage-bigtable/src/lib.rs

```diff
@@ -796,7 +796,11 @@ impl LedgerStorage {
             Some(before_signature) => {
                 let TransactionInfo { slot, index, .. } = bigtable
                     .get_bincode_cell("tx", before_signature.to_string())
-                    .await?;
+                    .await
+                    .map_err(|err| match err {
+                        bigtable::Error::RowNotFound => Error::SignatureNotFound,
+                        _ => err.into(),
```

Comment:
Something similar was added to `get_signature_status` in https://github.com/anza-xyz/agave/commit/71e9958e061493d7545bd28d4ac7a85aaed6ffbb but we could use it here, to disambiguate between ‘couldn't find the thing’ and ‘o no bigtable down bad.’